index.html

```diff
@@ -101,9 +101,16 @@ <h3>Finalize</h3>
                     <h2>Lorem ipsum dolor sit</h2>
                     <p>amet consetetur sadipscing elitr</p>
                   </div>
-                  <button class="btn-zoom">
-                    <img src="assets/img/magnifying-glass.png" alt="" />
-                  </button>
+                  <a
+                    data-fslightbox="lightbox-1"
+                    href="https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-1.jpg"
```

Comment:
As discussed, let's try with relative links

```suggestion
                    href="assets/img/gallery-img-1.jpg"
```

Reply:
Done (not working locally now).